src/main/java/net/sf/jabref/gui/groups/AutoGroupDialog.java

```diff
@@ -73,6 +74,8 @@
     private final JabRefFrame frame;
     private final BasePanel panel;
 
+    private static final LatexToUnicodeFormatter FORMATTER = new LatexToUnicodeFormatter();
```

Comment:
I would make the formatter a normal variable (initialized directly before the for loop below) instead of static final.